diff -rupN ecasound-2.9.3.old/configure.ac ecasound-2.9.3/configure.ac
--- ecasound-2.9.3.old/configure.ac	2020-01-11 13:11:38.000000000 +0100
+++ ecasound-2.9.3/configure.ac	2021-06-21 18:30:32.145397136 +0200
@@ -1136,7 +1136,7 @@ AC_ARG_ENABLE(liblilv,
     esac
   ],
   [
-    PKG_CHECK_MODULES([LIBLILV], [lv2core lilv-0 > 0.5.0 ], HAVE_LIBLILV=yes, HAVE_LIBLILV=no)
+    PKG_CHECK_MODULES([LIBLILV], [lv2 lilv-0 > 0.5.0 ], HAVE_LIBLILV=yes, HAVE_LIBLILV=no)
     enable_liblilv=${HAVE_LIBLILV}
     AC_MSG_RESULT(${enable_liblilv})
   ]
