diff -rupN ecasound-2.9.0.old/pyecasound/ecacontrol.py ecasound-2.9.0/pyecasound/ecacontrol.py
--- ecasound-2.9.0.old/pyecasound/ecacontrol.py	2012-04-15 22:35:35.000000000 +0200
+++ ecasound-2.9.0/pyecasound/ecacontrol.py	2013-05-04 22:17:00.320790580 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 """Native python ECI (ecasound control interface) implementation
 
    Can be used to replace the C implementation 'pyecasound.so'.
diff -rupN ecasound-2.9.0.old/pyecasound/eci.py ecasound-2.9.0/pyecasound/eci.py
--- ecasound-2.9.0.old/pyecasound/eci.py	2007-08-25 15:52:26.000000000 +0200
+++ ecasound-2.9.0/pyecasound/eci.py	2013-05-04 22:17:18.445206276 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 """
 # eci.ECI -- A higher-level interface to pyeca.
 # Copyright 2001 Eric S. Tiedemann (est@hyperreal.org)
diff -rupN ecasound-2.9.0.old/pyecasound/pyeca.py ecasound-2.9.0/pyecasound/pyeca.py
--- ecasound-2.9.0.old/pyecasound/pyeca.py	2012-04-15 20:51:29.000000000 +0200
+++ ecasound-2.9.0/pyecasound/pyeca.py	2013-05-04 22:17:33.618717110 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/python
 """Wrapper module which loads pyecasound
 (python module for Ecasound Control Interface).
 
diff -rupN ecasound-2.9.0.old/rubyecasound/ecasound.rb ecasound-2.9.0/rubyecasound/ecasound.rb
--- ecasound-2.9.0.old/rubyecasound/ecasound.rb	2007-08-25 15:52:26.000000000 +0200
+++ ecasound-2.9.0/rubyecasound/ecasound.rb	2013-05-04 22:18:14.700392736 +0200
@@ -1,3 +1,4 @@
+#!/usr/bin/ruby
 # This is a native implementation of Ecasound's control interface for Ruby.
 # Copyright (C) 2003 - 2004  Jan Weil <jan.weil@web.de>
 # 
diff -rupN ecasound-2.9.0.old/rubyecasound/stresstest.rb ecasound-2.9.0/rubyecasound/stresstest.rb
--- ecasound-2.9.0.old/rubyecasound/stresstest.rb	2007-08-25 15:52:26.000000000 +0200
+++ ecasound-2.9.0/rubyecasound/stresstest.rb	2013-05-04 22:18:30.038898268 +0200
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 
 # -----------------------------------------------------------------------
 # Runs a stress test using the ruby-ecasound ControlInterface
