diff -rupN ecasound-2.9.3.old/libecasound/audioio-forked-stream.cpp ecasound-2.9.3/libecasound/audioio-forked-stream.cpp
--- ecasound-2.9.3.old/libecasound/audioio-forked-stream.cpp	2011-05-21 18:00:39.000000000 +0200
+++ ecasound-2.9.3/libecasound/audioio-forked-stream.cpp	2021-12-30 13:39:44.404774225 +0100
@@ -207,7 +207,7 @@ void AUDIO_IO_FORKED_STREAM::fork_child_
   else {
     int fpipes[2];
     if (pipe(fpipes) == 0) {
-      sigkill_sent_rep = false;
+      sigterm_sent_rep = false;
       pid_of_child_rep = fork();
       if (pid_of_child_rep == 0) { 
 	// ---
@@ -273,7 +273,7 @@ void AUDIO_IO_FORKED_STREAM::fork_child_
 
   init_state_before_fork();
 
-  sigkill_sent_rep = false;
+  sigterm_sent_rep = false;
   pid_of_child_rep = fork();
   if (pid_of_child_rep == 0) { 
     // ---
@@ -330,7 +330,7 @@ void AUDIO_IO_FORKED_STREAM::fork_child_
 
   int fpipes[2];
   if (pipe(fpipes) == 0) {
-    sigkill_sent_rep = false;
+    sigterm_sent_rep = false;
     pid_of_child_rep = fork();
     if (pid_of_child_rep == 0) { 
       // ---
@@ -384,7 +384,7 @@ void AUDIO_IO_FORKED_STREAM::fork_child_
  * it attemts to terminate the child anyways, but the child's 
  *  state is not known exactly when function returns.
  *
- * @param force if true, client is terminated with SIGKILL,
+ * @param force if true, client is terminated with SIGTERM,
  *              which guarantees that it terminates (but 
  *              possibly without going through normal 
  *              exit procedure); should be avoided especially
@@ -405,15 +405,15 @@ void AUDIO_IO_FORKED_STREAM::clean_child
 
   if (pid_of_child_rep > 0 && 
       force == true) {
-    if (sigkill_sent_rep != true) {
+    if (sigterm_sent_rep != true) {
       ECA_LOG_MSG(ECA_LOGGER::system_objects, 
-		  "Sending SIGKILL to child process related to: "
+		  "Sending SIGTERM to child process related to: "
 		  + object_rep);
-      kill(pid_of_child_rep, SIGKILL);
-      sigkill_sent_rep = true;
+      kill(pid_of_child_rep, SIGTERM);
+      sigterm_sent_rep = true;
     }
     else {
-      /* SIGKILL already sent once for this process, don't send it again */
+      /* SIGTERM already sent once for this process, don't send it again */
       pid_of_child_rep = -1;
     }
   }
diff -rupN ecasound-2.9.3.old/libecasound/audioio-forked-stream.h ecasound-2.9.3/libecasound/audioio-forked-stream.h
--- ecasound-2.9.3.old/libecasound/audioio-forked-stream.h	2009-02-01 19:08:41.000000000 +0100
+++ ecasound-2.9.3/libecasound/audioio-forked-stream.h	2021-12-30 13:37:37.293043101 +0100
@@ -22,7 +22,7 @@ class AUDIO_IO_FORKED_STREAM : public AU
   int pid_of_child_rep;
   int fd_rep;
   bool last_fork_rep;
-  bool sigkill_sent_rep;
+  bool sigterm_sent_rep;
   std::string tmpfile_repp;
   bool tmp_file_created_rep;
   bool use_named_pipe_rep;
@@ -74,7 +74,7 @@ public:
     pid_of_child_rep(-1),
     fd_rep(0),
     last_fork_rep(false),
-    sigkill_sent_rep(false),
+    sigterm_sent_rep(false),
     tmp_file_created_rep(false),
     use_named_pipe_rep(false) { }
   virtual ~AUDIO_IO_FORKED_STREAM(void);
