diff -rupN ecasound-2.9.3.old/libecasound/Makefile.am ecasound-2.9.3/libecasound/Makefile.am
--- ecasound-2.9.3.old/libecasound/Makefile.am	2021-06-21 18:39:18.000000000 +0200
+++ ecasound-2.9.3/libecasound/Makefile.am	2021-06-21 18:46:03.543630438 +0200
@@ -34,7 +34,7 @@ else
 eca_libadd =  	$(top_builddir)/libecasound/plugins/libecasound_plugins.la $(ECA_S_EXTRA_LIBS)
 endif
 
-eca_ldflags = -version-info @LIBECASOUND_VERSION@:0:@LIBECASOUND_VERSION_AGE@
+eca_ldflags = -version-info @LIBECASOUND_VERSION@:0:@LIBECASOUND_VERSION_AGE@ ../kvutils/libkvutils.la
 
 if ECA_AM_DEBUG_MODE
 libecasound_tester_libs =	$(top_builddir)/libecasound/libecasound_debug.la \
